src/test/java/roomescape/controller/ReservationControllerTest.java

```diff
@@ -0,0 +1,143 @@
+package roomescape.controller;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+class ReservationControllerTest {
+
+    @DisplayName("Reservation 목록 내용 갯수를 검사한다")
+    @Test
+    void reservationTest() {
+        RestAssured.given().log().all()
+                .when().get("/reservations")
+                .then().log().all()
+                .statusCode(200)
+                .body("size()", is(0));
+    }
+
+    @Nested
+    class ReservationInsertTest {
+
+        @BeforeEach
+        void setUp() {
+            Map<String, Object> reservationTime = new HashMap<>();
+            reservationTime.put("startAt", "10:00");
+
+            RestAssured.given().log().all()
+                    .contentType(ContentType.JSON)
+                    .body(reservationTime)
+                    .when().post("/times")
+                    .then().log().all()
+                    .statusCode(200);
+        }
```

Comment:
아래와 같이 객체를 넘기는 방법도 있습니다 :)

```suggestion
        @BeforeEach
        void setUp() {
            ReservationTimeCreateRequestDto request = new ReservationTimeCreateRequestDto(LocalTime.of(10, 0));

            RestAssured.given().log().all()
                    .contentType(ContentType.JSON)
                    .body(request)
                    .when().post("/times")
                    .then().log().all()
                    .statusCode(200);
        }
```

Reply:
객체로 바로 넘기는 방법도 있군요! 반영하였습니다.